inout/Cargo.toml

```diff
@@ -14,10 +14,3 @@ readme = "README.md"
 [dependencies]
 block-padding = { version = "0.4.0-rc.0", path = "../block-padding", optional = true }
 hybrid-array = "0.2.0-rc.10"
-
-[features]
-std = ["block-padding/std"]
-
-[package.metadata.docs.rs]
-all-features = true
```

Comment:
Looks like we still need the `all-features = true` here because of the `block-padding` feature.

I think `--cfg docsrs` might be set automatically now though?

Reply:
>I think --cfg docsrs might be set automatically now though?

It would be great, if true. We should eventually remove it from other crates then.